CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.1.0] - 2025-05-08
```

Comment:
If I redo Issue #151, and point it at this branch, would I be able to slip it into the 2.1.0 release so I don't have to do a 2.1.1 release soon after?

If you want this out sooner, that's cool too.

Reply:
EDIT* I can probably get this done by tomorrow, just have to switch contexts

Reply:
slipping it in to 2.1.0 works for me, thanks!